data/binding/binding.go

```diff
@@ -15,6 +15,12 @@ type DataItem interface {
 	RemoveListener(DataListener)
 }
 
+// DataItemFromVariable is used by any binding that is backed by a source variable.
```

Comment:
I think we could come up with a better name, maybe DataPointer, or DataAddress?

Reply:
Similarly; UpdateFromPointer, UpdateFromAddress?

Reply:
Or maybe we should follow the 'On' prefix; OnValueUpdated

Reply:
I will think more about the top two.
On item 3 certainly not - "On..." is for user registered callbacks, this however is equivalent of `Refresh()` as the developer is telling the toolkit something...

Reply:
> DataPointer, DataReference, or DataAddress

I'm not keen on pointer or reference as they can mean specific things which may or may not technically hold through the implementation. Another name in the ring, `DataItemFromSource` or `DataItemWithSource`?

Reply:
I'm not sure about `<Data>FromPointer`, I thought we'd agreed on `<Data>Pointer`?

Reply:
Yes, I added that to address the concerns about needing a `NewDataPointer` which didn't seem desirable. Rolled that back as-per Slack conversation.